src/main/kotlin/no/nav/helse/fritakagp/koin/Fakes.kt

```diff
@@ -41,7 +41,9 @@ import java.time.LocalDateTime
 fun Module.mockExternalDependencies() {
     single { MockOAuth2Server().apply { start(port = 6668) } }
     single {
-        mockk<AuthClient> {
+        spyk(
+            AuthClient("token-endpoint", "token-exchange-endpoint", "token-introspection-endpoint")
+        ) {
```

Comment:
Endret noen extension functions til vanlige klassefunksjoner, så da blir denne en "spy" for å unngå å mocke de "nye" klassefunksjonene.